modules/internal/OwnedObject.chpl

```diff
@@ -22,13 +22,11 @@
 module OwnedObject {
   use ChapelStandard;
 
-  // Ideally, this can be marked with nodoc and the doc put at the module level
-  //   since owned isn't really a 'record' or a 'type'
-  // But that prevents us from referencing a "top-level" owned reference
-  // If we one day support :noindexentry:, that could be applied at the module
-  //   level (instead of :noindex:)
-  // And then we could do :mod:`owned <OwnedObject>`
-  // For now, `fixInternalDocs.sh` replaces `.. record:: owned` with `.. type:: owned`
+  // TODO: Ideally, this can be marked with nodoc and the doc put at the module
```

Comment:
In the spirit of docs week, would it be possible to open a public issue for this or find an existing one?